charts/dbildungs-iam-server/values.yaml

```diff
@@ -133,6 +133,9 @@ backend:
       subPath: secrets-json
       name: secret-volume
   extraVolumeMounts: []
+  config: 
+    alternativeConfigJson: false
+    alternativeConfigFile: {}
```

Comment:
I think this should be called something like alternativeConfigFileContent to make its usage more clear